Library/Homebrew/formula.rb

```diff
@@ -3152,8 +3151,7 @@ def patch(strip = :p1, src = nil, &block)
     #
     # @deprecated Please use {Homebrew::Service.require_root} instead.
     def plist_options(options)
-      # TODO: Deprecate
-      # odeprecated "plist_options", "service.require_root"
+      odeprecated "plist_options", "service.require_root"
```

Comment:
I guess for this we want to deprecate setting this information now but not getting this information.

Reply:
How can I make that distinction in the deprecation?

Reply:
@SMillerDev You can't. It needs to be done programmatically so you only call `odeprecated in that situation. It looks like this method maybe only does set though anyway?

Reply:
Yeah, I think so.